src/sentry/apidocs/parameters.py

```diff
@@ -271,3 +271,46 @@ class ReplayParams:
         type=OpenApiTypes.UUID,
         description="""The ID of the replay you'd like to retrieve.""",
     )
+
+
+class IntegrationParams:
+    PROVIDER_KEY = OpenApiParameter(
+        name="providerKey",
+        location="query",
+        required=False,
+        type=str,
+        description="""Specific integration provider to filter by. For example, `slack`. See [Integrations Documentation](/product/integrations/) for an updated list of providers.""",
+    )
+    FEATURES = OpenApiParameter(
+        name="features",
+        location="query",
+        required=False,
+        type=str,
+        many=True,
+        description="""Integration features to filter by. See [Integrations Documentation](/product/integrations/) for an updated list of features. Current available ones are:
+         - alert-rule
+         - chat-unfurl
+         - codeowners
+         - commits
+         - data-forwarding
+         - deployment
+         - enterprise-alert-rule
+         - enterprise-incident-management
+         - incident-management
+         - issue-basic
+         - issue-sync
+         - mobile
+         - serverless
+         - session-replay
+         - stacktrace-link
+         - ticket-rules
+         """,
+    )
+    INCLUDE_CONFIG = OpenApiParameter(
+        name="includeConfig",
+        location="query",
+        required=False,
+        type=bool,
+        description="""Set to `True` to include integration configurations fetched from third-party
+            in the response. Note that this can add several seconds to the response time.""",
```

Comment:
```suggestion
        description="""Specify `True` to fetch third-party integration configurations. Note that this can add several seconds to the response time.""",
```